cli/src/generate/grammar_files.rs

```diff
@@ -261,9 +261,19 @@ pub fn generate_grammar_files(
             generate_file(path, LIB_RS_TEMPLATE, language_name)
         })?;
 
-        missing_path(path.join("build.rs"), |path| {
-            generate_file(path, BUILD_RS_TEMPLATE, language_name)
-        })?;
+        missing_path_else(
+            path.join("build.rs"),
+            |path| generate_file(path, BUILD_RS_TEMPLATE, language_name),
+            |path| {
+                let build_rs =
+                    fs::read_to_string(path).with_context(|| "Failed to read build.rs")?;
+                if !build_rs.contains("/utf-8") {
+                    eprintln!("Updating build.rs with the /utf-8 flag for Windows");
+                    generate_file(path, BUILD_RS_TEMPLATE, language_name)?;
```

Comment:
It might be better to print a warning here instead of replacing the file.

Reply:
it'd be annoying for users that don't care about the rust grammar, but have this warning printed every time they run generate

Reply:
But users that do care about the rust bindings may have to edit them again.

Reply:
what if we just place it after `println!("cargo:rerun-if-changed={}", parser_path.to_str().unwrap());` instead of generating from the template?